apis/src/main/java/xyz/bookself/controllers/user/ShelfNameDTO.java

```diff
@@ -0,0 +1,12 @@
+package xyz.bookself.controllers.user;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import java.util.Set;
+
+@Getter
+@Setter
+public class ShelfNameDTO {
+    private String newListName;
```

Comment:
You can use the `@NotBlank` annotation here. Then Spring will automatically send back a 422 if the value is null or blank.